app/models/miq_queue.rb

```diff
@@ -174,6 +174,12 @@ def self.get(options = {})
     result
   end
 
+  # This are the queue calls related to worker management which
+  # might not be needed once we use kubernetes for worker/pod management
+  def self.put_deprecated(*args)
```

Comment:
@jrafanie We are in the process of bucketing (cataloging) the MiqQueue.put methods based on the use case, which allows us to mark ones we think should exist in the current version but will get deprecated when me move to the new architecture.